src/sql/workbench/services/query/common/gridDataProvider.ts

```diff
@@ -133,7 +133,7 @@ export async function copySelectionToClipboard(clipboardService: IClipboardServi
 		const getMessageText = (): string => {
 			return nls.localize('gridDataProvider.loadingRowsInProgress', "Loading the rows to be copied ({0}/{1})...", processedRows, rowCount);
 		};
-		let resultString = '';
+		let headerString = '';
```

Comment:
nit: Instead of making a whole separate var and copying that into the array you create below I'd suggest just creating the array here and push the header to it directly on line 145.

Reply:
Yeah, I guess I could do that but I wanted the logic of string-join to be separated from the logic of creating the data rows.